--- rad_eap_test.orig	2019-03-01 14:00:13.000000000 +0000
+++ rad_eap_test	2019-03-01 14:00:31.000000000 +0000
@@ -274,7 +274,7 @@
 #cat $CONF
 
 # address may be address or ip address
-IP=`echo $ADRESS | grep --regexp="^[[:digit:]+\.[:digit:]+\.[:digit:]+\.[:digit:]]"`
+IP=`echo $ADRESS | egrep "^[[:digit:]+\.[:digit:]+\.[:digit:]+\.[:digit:]]"`
 if [ -z "$IP" ]; then 
   #IP=`host $ADRESS | tail -n 1 | awk '{print $3}'`
   IP=`dig +noall +answer +search $ADRESS | tr \\\t ' ' | grep ' IN A ' | sed "s/.* IN A //"`
